src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -651,6 +666,18 @@ private static bool TryGetIntQueryParameterValue(NameValueCollection queryString
             return int.TryParse(value, out intValue);
         }
 
+        private static bool TryGetTimeSpanQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName, out TimeSpan? timeSpanValue)
+        {
+            timeSpanValue = null;
+            string value = queryStringNameValueCollection[queryParameterName];
+            if (value != null)
+            {
+                timeSpanValue = TimeSpan.FromSeconds(double.Parse(value));

```

Comment:
It's not clear to me why a TryGetMethod is better than just returning a nullable TimeSpan, as the `HasValue` on nullable types serves the same purpose as the return type from this method.

Reply:
I did this mainly to match how we are getting query parameters elsewhere. I think this method was just accidentally left behind when the TryGets were added last year.

Reply:
Fair enough. Consistency is probably more important than orthodoxy here.